taco-tuesday-rest-api/src/main/java/com/muskopf/tacotuesday/domain/Employee.java

```diff
@@ -14,20 +19,24 @@
     @GeneratedValue
     private Integer id;
 
-    @Column
+    @CreationTimestamp
+    private Instant createdAt;
+
+    @Column(nullable = false)
     private String firstName;
-    @Column
+    @Column(nullable = false)
     private String lastName;
+    @Column(unique = true, nullable = false)
+    private String slackId;
     @Column
     private String nickName;
-    @Column
-    private String slackId;
 
     @OneToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
     @JoinColumn(name = "api_key_id", referencedColumnName = "id")
     private ApiKey apiKey;
 
-    @OneToMany(mappedBy = "employee", cascade = CascadeType.PERSIST)
+    @OneToMany(mappedBy = "employee", cascade = CascadeType.MERGE, fetch = FetchType.LAZY)
+    @JsonIgnore
```

Comment:
Don't want these coming back in a `GET /employees`.
If someone wants an employee's orders they should use the `GET /order/individual` endpoint after the resolution of issue #25